protos/in_toto_attestation/predicates/vuln/v0.1/vuln.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package in_toto_attestation.predicates.vuln.v01;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/in-toto/attestation/go/predicates/vuln/v1";
+option java_package = "io.github.intoto.attestation.predicates.vuln.v1";
+
+// Validation of all fields is left to the users of this proto.
+message Vuln {
+  Scanner scanner = 1;
+  ScanMetadata scan_metadata = 2;
+}
+
+message Scanner {
+  string uri = 1;
+  optional string version = 2;
+  VulnDatabase database = 3;
+  repeated Result result = 4;
+}
+
+message VulnDatabase {
+  optional string uri = 1;
+  optional string version = 2;
+  google.protobuf.Timestamp last_update = 3;
+}
+
+message Result {
+  string id = 1;
+  repeated Severity severity = 2;
+
+  message Severity {
+    string method = 1;
+    string score = 2;
+  }
+
+  repeated google.protobuf.Struct annotations = 3;
```

Comment:
i think the spec needs some clarification on this struct (i created https://github.com/in-toto/attestation/issues/390 to ask for some clarification)